backend/sql.go

```diff
@@ -8,18 +8,156 @@ import (
 func CreateSqlTables() {
 	if _, err := db.Exec(`BEGIN;
 
-CREATE TABLE IF NOT EXISTS users (
-	username VARCHAR(16) NOT NULL UNIQUE,
-	password VARCHAR(100) NOT NULL,
-	email VARCHAR(319) NOT NULL UNIQUE,
-	id UUID NOT NULL PRIMARY KEY,
-	created_at TIMESTAMP NOT NULL DEFAULT NOW(),
-	verified BOOLEAN NOT NULL DEFAULT FALSE);
-
 CREATE TABLE IF NOT EXISTS tokens (
-	token VARCHAR(128) NOT NULL PRIMARY KEY,
-	created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
-	user_id UUID NOT NULL REFERENCES users(id));
+    token VARCHAR(128) NOT NULL PRIMARY KEY,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
```

Comment:
```suggestion
    created_at DATETIME NOT NULL DEFAULT NOW(),
```
TIMESTAMPTZ was just copied from PostgreSQL, this doesn't work in MySQL